src/modal/component/accordion/Accordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+
+import AccordionItem from "./item/AccordionItem";
+import {AccordionData} from "./util/accordionTypes";
+
+interface AccordionProps {
+  accordionData: AccordionData[];
+}
+
+function Accordion({accordionData}: AccordionProps) {
+  const [clicked, setClicked] = useState("0");
+
+  return (
+    <ul className={"accordion"}>
```

Comment:
Let's use `pera-wallet` prefix, generic classNames may cause style conflict in projects
```suggestion
    <ul className={"pera-wallet-accordion"}>
```